### PR TITLE
add Design Pattern Go

### PR DESCRIPTION
**Pull request by egaprsty**

### CODE REVIEW

**Review by egaprsty — COMMENTED** [review]
New Add (Rust)

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ This list aims to inspire others and hopefully give encouragement for individual
 - [Excel2Json](https://github.com/FerdinaKusumah/excel2json) - Convert your excel or excel datasource to json easily.
 - [Teleport](https://github.com/muhammadmuhlas/teleport) - A simple tool based on Go to migrate git repository.
 - [PHC Crypto](https://github.com/aldy505/phc-crypto) - Password hashing algorithm based on PHC standard, inspired by Upash package from Node.js.
-- [Unknwon](https://github.com/unknwon/build-web-application-with-golang_EN) - Build Web Application with Go and learn basic.
+- [Imre Nagi](https://github.com/imrenagi/design-pattern) - Design pattern with Go
```
Comment (troke12) [inline README.md:160]:
```suggestion
- [Design Pattern](https://github.com/imrenagi/design-pattern) - Design pattern with Go
```

Use format : 
`[Repository Name](url)`

File: README.md [hunk 2]
```diff
@@ -167,6 +167,7 @@ This list aims to inspire others and hopefully give encouragement for individual
 ## Rust
 
 - [Fido](https://github.com/codenoid/Fido) - Distributed Storage, Easily distribute your data accross disk/node.
+- [Evil Factory Labs](https://github.com/evilfactorylabs/belajar-rust) - Learn Rust From Scratch
```
Comment (troke12) [inline README.md:170]: same with this

File: README.md (hunk 1 above)
Comment (egaprsty) [reply README.md]: change format done

File: README.md (hunk 2 above)
Comment (egaprsty) [reply README.md]: done

**Review by troke12 — APPROVED** [review]

**Comment by egaprsty** [comment]
thx a lot

**Pull request merged**